code/datums/outfits/ert/fsf_ert.dm

```diff
@@ -0,0 +1,118 @@
+/datum/outfit/admin/ert/fsf_ert
+    name = "Free Solarian Fleets Marine"
+
+    uniform = /obj/item/clothing/under/rank/fatigues/marine
+    shoes = /obj/item/clothing/shoes/swat/ert
+    gloves = /obj/item/clothing/gloves/swat/ert
+    belt = /obj/item/storage/belt/military
+    back = /obj/item/storage/backpack/satchel_norm
+    head = /obj/item/clothing/head/helmet/space/void/sol
```

Comment:
if you're giving them a voidsuit, there's a proc i added to lancers that automatically sets up their voidsuits

Reply:
though it needs you to put an oxygen tank in their suit slot, so i don't think it's needed here